### PR TITLE
Version Information

### PR DESCRIPTION
**Pull request by waxenegger**
see: https://trello.com/c/GYwXm4BY/16-add-about

@jburel  I know we said this would not be needed for this upcoming release, and feel free to not include it, but it was a minor thing to do and apart from the last commit (display of version) the code will be needed to hand in the version information regardless of how it ends up being displayed eventually.

### CODE REVIEW

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [iviewer-merge#374](https://ci.openmicroscopy.org/job/iviewer-merge/374/). See the [console output](https://ci.openmicroscopy.org/job/iviewer-merge/374/consoleText) for more details.
Possible conflicts:
  - PR #93 waxenegger 'add viewer method to turn off smoothing'
    - src/utils/constants.js
  - PR #96 waxenegger 'luts janelia'
    - src/utils/constants.js
  - PR #99 waxenegger 'open with images'
    - src/utils/constants.js
  - Upstream changes
    - src/utils/constants.js

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [iviewer-merge#375](https://ci.openmicroscopy.org/job/iviewer-merge/375/). See the [console output](https://ci.openmicroscopy.org/job/iviewer-merge/375/consoleText) for more details.
Possible conflicts:
  - PR #93 waxenegger 'add viewer method to turn off smoothing'
    - src/utils/constants.js
  - PR #96 waxenegger 'luts janelia'
    - src/utils/constants.js
  - PR #99 waxenegger 'open with images'
    - src/utils/constants.js
  - Upstream changes
    - src/utils/constants.js

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [iviewer-merge#376](https://ci.openmicroscopy.org/job/iviewer-merge/376/). See the [console output](https://ci.openmicroscopy.org/job/iviewer-merge/376/consoleText) for more details.
Possible conflicts:
  - PR #99 waxenegger 'open with images'
    - src/utils/constants.js

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [iviewer-merge#377](https://ci.openmicroscopy.org/job/iviewer-merge/377/). See the [console output](https://ci.openmicroscopy.org/job/iviewer-merge/377/consoleText) for more details.
Possible conflicts:
  - PR #99 waxenegger 'open with images'
    - src/utils/constants.js

**Comment by waxenegger** [comment]
--exclude

**Comment by jburel** [comment]
@waxenegger if you have a chance, could you fix the merge conflict and remove the exclude label?

**Comment by mtbc** [comment]
iviewer doesn't seem to be working at all for me today:

iviewer

File: src/app/header.html [hunk 1]
```diff
@@ -18,6 +18,9 @@
 
 <template>
     <nav class="navbar navbar-default navbar-fixed-top fixed-header">
-        <div class="fixed-header-text">${short_image_name}</strong></div>
+        <div class="fixed-header-text">
+           <span>${short_image_name}<span>
```
Comment (mtbc) [inline src/app/header.html]: probably meant to close that tag

**Comment by jburel** [comment]
@mtbc happy with the last commit?

**Comment by mtbc** [comment]
Yes, I am.

**Pull request merged**